src/main/kotlin/solve/utils/structures/DoublePoint.kt

```diff
@@ -0,0 +1,16 @@
+package solve.utils.structures
+
+import kotlin.math.pow
+import kotlin.math.sqrt
+
+data class DoublePoint(val x: Double, val y: Double) {
```

Comment:
> Maybe make it and Size generic and use one class everywhere

As it is turned out, it is impossible to create a generic class that uses numbers because the Number parent class does not have needed arithmetic methods